package.json

```diff
@@ -21,6 +21,7 @@
   "devDependencies": {
     "@faker-js/faker": "7.4.0",
     "chai": "^4.3.6",
+    "crypto": "^1.0.1",
```

Comment:
@beblicarl its inbuilt, no need to add a dependency

Reply:
yes please